Localize/lang/strings.json

```
@@ -2295,6 +2315,7 @@
   "isnFBi": "Elements and mappings not connected to a target element are removed.",
   "ixhS47": "Returns the start of the day to a string timestamp passed in",
```

See ["time stamp" - Microsoft Style Guide](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/t/time-stamp).

```suggestion
  "ixhS47": "Returns the start of the day for the passed-in string timestamp.",
```